models/models.js

```diff
@@ -82,11 +88,15 @@ exports.checkAuthorExists = (username) => {
         });
     };
 
-exports.updateArticles = (article_id, new_votes) => {
+exports.updateArticles = (article_id, inc_votes) => {
+    if(!inc_votes){
+        return Promise.reject({ status: 400, msg: 'Invalid Input' })
+    }
```

Comment:
This can be dealt with as a custom error, but actually psql will handle it for you. It's gonna be a 23502 error, can add the if statement in your error handler and also send another message "required key missing" rather than "Invalid Input"